son-gtklic/docs/swagger.yaml

```diff
@@ -0,0 +1,547 @@
+# this is an example of the Uber API
+# as a demonstration of an API spec in YAML
+swagger: '2.0'
+info:
+  title: Sonata's License Manager
+  description: Manage licenses Sonata services.
+  version: "1"
+# the domain of the service
+host: sonata-vm1.aws.atnog.av.it.pt
+# array of all schemes that your API supports
+schemes:
+  - http
+# will be prefixed to all paths
+basePath: /api/v1
```

Comment:
Ok, returning the version in our responses is easily doable.

So, you are also requesting to read the X-Version header, in the beginning of every endpoint function and validate the version (if version == 1.0 do something; else if version == 2.0 do other something)? Because this way, every version in the future would need to be implemented in the same file. Can you explain further this idea of versioning?